examples/deployments/deployments_server/openai/config.yaml

```
@@ -6,6 +6,9 @@ endpoints:
       name: gpt-3.5-turbo
       config:
         openai_api_key: $OPENAI_API_KEY
+    limit:
+      renewal_period: "minute"
+      calls: 1
```

```suggestion
      calls: 10
```

I often use this config. 1 seems inconvenient :)